packages/terra-badge/tests/nightwatch/BadgeDefault.jsx

```diff
@@ -3,7 +3,7 @@ import Badge from '../../lib/Badge';
 
 
 const BadgeDefault = () => (
-  <Badge text="Default" />
+  <Badge text="Default" id="default-badge" />
```

Comment:
If you haven't already you should add your name to the contributors file: https://github.com/cerner/terra-core/blob/master/CONTRIBUTORS.md

Reply:
Done

https://github.com/cerner/terra-core/pull/803/commits/ab1f428353a8f401e0f45c576075e6ba1e25f22f

Reply:
Welcome to the club.